### ISSUE TITLE
connection failed with https and error 503 with http

### ISSUE THREAD

**Issue by uportalis** [loomio/loomio-deploy#3]
Hi

Following the installation procedure I got the following problems:
* Accessing my server in HTTPS gives a "connection failed" error on my web browser (and nothing appear in the logs)
* Accessing in HTTP give error 503.

Please also note that:
* My server is not accessible from Internet (so certificate stuff with Let's Encrypt will not work, as far as I understand)
* docker-compose.yml file contains WEBHOOK_URL=http://loomio:3000/email_processor/. Should I change it?

By the way, I do not need HTTPS (my server is accessible from a VPN only). Can I disable it?

Please see the attached logs:
[docker-compose logs.txt](https://github.com/loomio/loomio-deploy/files/277503/docker-compose.logs.txt)

Rgds,
Louis

**Comment by uportalis** [comment]
FYI, the only thing I did not do from the install instructions is the MX configuration.

**Comment by uportalis** [comment]
When connecting to HTTP, the server redirects to HTTPS.
This redirection and error 503 is documented on https://github.com/jwilder/nginx-proxy.
The error 503 occurs when no suitable certificate can be found.

I finally solved my problem like this:
- I generated two self signed certificates using openssl and name them mydomain.crt/mydomain.key and faye.mydomain.crt/faye.mydomain.key
- I commented the letsencrypt part in docker-compose.yml

I think this bug report can be closed.
